### ISSUE TITLE
classification of image with no objects but having class label as 0

### ISSUE THREAD

**Issue by sramakintel** [kuangliu/pytorch-retinanet#77]
If an image has just background (no objects in entire image,so no need of bounding boxes) with class label 0, will this code give the correct classification as class 0 or should I tweak the loss function?
